src/Nethermind/Nethermind.Db/DbOnTheRocks.cs

```diff
@@ -31,18 +31,23 @@ namespace Nethermind.Db
     public abstract class DbOnTheRocks : IDb, IDbWithSpan
     {
         private static readonly ConcurrentDictionary<string, RocksDb> DbsByPath = new ConcurrentDictionary<string, RocksDb>();
-        private readonly RocksDb _db;
-        private WriteBatch _currentBatch;
-        private WriteOptions _writeOptions;
+        internal readonly RocksDb Db;
+        internal WriteBatch CurrentBatch;
```

Comment:
for testing?

Reply:
Nope, ColumnDb, made it to readonly properties